cogs/deepfry.py

```diff
@@ -0,0 +1,65 @@
+from io import BytesIO
+import discord
+import requests
+from PIL import Image, ImageOps, ImageEnhance
+from discord.ext import commands
+from cogs.base import BaseCog
```

Comment:
I don't know for sure, but I don't these imports are in the right order

Reply:
Did we ever determine the standard order we should follow for imports and if this is following them?